Cargo.toml

```diff
@@ -49,7 +49,7 @@ sqlx = {version = "0", features = [
   "macros",
   "runtime-tokio-rustls"
 ]}
-helium-proto = {git = "https://github.com/helium/proto", branch = "master", features = ["services"]}
+helium-proto = {git = "https://github.com/helium/proto", branch = "andymck/rename-lora-to-iot", features = ["services"]}
```

Comment:
```suggestion
helium-proto = {git = "https://github.com/helium/proto", branch = "master", features = ["services"]}
```

Reply:
looks like the latest proto PR build had failed the CPP pipeline for a repeating but infrequent api rate limit problem; i re-kicked that build and it passed immediately, so it should be good to merge now, followed by updating this PR and merging